packages/FrontC/FrontC.3.4.3/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+authors: "Hugues Cassé <casse@irit.fr>"
+homepage: "https://github.com/BinaryAnalysisPlatform/FrontC"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/FrontC/issues"
+dev-repo: "git+https://github.com/BinaryAnalysisPlatform/FrontC"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+license: "LGPL-2.1-only"
+depends: [
+  "ocaml"
+]
+install: [
+  [make "install" "PREFIX=%{lib}%" "OCAML_SITE=%{lib}%"]
```

Comment:
I'm not seeing any `build` section in this package. Are there nothing built? If so, please use it, putting everything in the `install` section will slow down concurrent installation for your users'

Reply:
good point! ALso, is there any sense in `FrontC.install` file? I have inherited it from the previous maintainers of FrontC and have no idea what it is meant for. Any guesses?